llama_stack/apis/eval/eval.py

```diff
@@ -74,14 +74,14 @@ class EvaluateResponse(BaseModel):
 
 
 class Eval(Protocol):
-    @webmethod(route="/eval/run", method="POST")
+    @webmethod(route="/eval/tasks/{task_id}/jobs", method="POST")
```

Comment:
```suggestion
    @webmethod(route="/eval/tasks/{task_id}/run", method="POST")
```
Can we change it to submit since this function basically "runs" an eval job?